### PR TITLE
Add registry entries for several plugins.

### PR DESCRIPTION
**Pull request by greschd**
These are my plugins for creation of tight-binding models from first-principles calculations:
- aiida-optimize
- aiida-bands-inspect
- aiida-symmetry-representation
- aiida-strain
- aiida-tbmodels
- aiida-tbextraction

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -208,5 +208,59 @@
...
         "plugin_info": "https://raw.githubusercontent.com/ltalirz/aiida-qeq/master/setup.json",
         "code_home": "http://github.com/ltalirz/aiida-qeq"
+    },
+    "optimize": {
+        "name": "aiida-optimize",
+        "entry_point": "optimize",
+        "state": "development",
+        "plugin_info": "https://raw.githubusercontent.com/greschd/aiida-optimize/master/setup.json",
+        "code_home": "https://github.com/greschd/aiida-optimize",
+        "documentation_url": "https://aiida-strain.readthedocs.io",
```
Comment (ltalirz) [inline plugins.json]: please fix url

**Comment by ltalirz** [comment]
Thanks, looks very interesting!

Before approving the PR: some of the plugin names are very general (optimize, strain, symmetry-representation).

Are the plugins as general as these names suggest, i.e. should they be/become the general solution for these types of problems?

Also mentioning @giovannipizzi

**Comment by greschd** [comment]
In general yes, but "strain" still has some ways to go -- it only implements III-V semiconductors, but the same interface could be used for other structures.

The "symmetry-representation" code refers to http://z2pack.ethz.ch/symmetry-representation/, but the AiiDA plugin currently only has the functionality to filter symmetries which are consistent with a given structure.

And "optimize" implements a general workchain which can be fed a "target" workchain and optimization engine to perform some form of parameter optimization. It currently has parameter sweeps, bisection and Nelder-Mead engines.

File: plugins.json [hunk 2]
```diff
@@ -215,7 +215,7 @@
         "state": "development",
         "plugin_info": "https://raw.githubusercontent.com/greschd/aiida-optimize/master/setup.json",
         "code_home": "https://github.com/greschd/aiida-optimize",
-        "documentation_url": "https://aiida-strain.readthedocs.io",
+        "documentation_url": "https://aiida-optimize.readthedocs.io",
         "pip_url": "aiida-strain"
```
Comment (ltalirz) [inline plugins.json]: there's a few other places, perhaps just search for aiida-strain

Comment (greschd) [reply plugins.json]: Oh yeah, and here I thought I'd checked it by eye...

**Comment by ltalirz** [comment]
@giovannipizzi Just waiting for your approval here for the names of the plugins...

**Comment by giovannipizzi** [comment]
Ok to approve. @greschd could you please make sure that the 'title' and 'description' in the setup.json are all descriptive enough, so people trying to understand what the package do understand it easily? You can check what appears in the compiled version of the registry and adapt accordingly (note that the compiled version will be changed only with a new commit here)

**Pull request merged**